CHANGELOG.rst

```diff
@@ -2,11 +2,20 @@
 Changelog
 =========
 
-Upgrade notes
--------------
+2.7.10 (2024-11-22)
+===================
+
+Periodic bugfix release
 
+* [:backend:`4732`] Fixed CSP issues for Expoints/other analytics.
+* [:backend:`4745`] Fixed missing registration variable to the Objects API with all
+  the attachment URLs.
 * [:backend:`4810`] Fixed uppercase component variable values turing lowercase.
   There are manual actions required.
```

Comment:
Please include the steps to execute too, see for example https://open-forms.readthedocs.io/en/stable/archive/changes-v2.html#id68